odl/operator/operator.py

```diff
@@ -2073,8 +2074,11 @@ def adjoint(self):
         if not self.is_linear:
             raise OpNotImplementedError('nonlinear operators have no adjoint')
 
-        # TODO: handle complex vectors
-        return self.operator.adjoint * self.vector
+        if self.vector.space.is_rn:
+            '''The complex conjugate of a real vector is the vector itself.'''
```

Comment:
Always use # for inline comments. 

What happens here is technically that you create a string with the text, and then never assign it to anything. This is a slight performance problem.

Comments "at the top" are special since we actually *want* to assign them to something, but that is done implicitly by python. You can check `obj.__doc__` to see where it was assigned, and find more information here https://www.python.org/dev/peps/pep-0257.

Anyway, fix that and then we go!